aztec/src/test/kotlin/org/wordpress/aztec/IndentTest.kt

```diff
@@ -0,0 +1,301 @@
+package org.wordpress.aztec
+
+import android.app.Activity
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.wordpress.aztec.source.SourceViewEditText
+import org.wordpress.aztec.toolbar.AztecToolbar
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = intArrayOf(23))
```

Comment:
Lint warning: should be replaced with array literal

```suggestion
@Config(sdk = [23])
```